src/Gui/Command.cpp

```diff
@@ -239,6 +258,12 @@ void Command::addTo(QWidget *pcWidget)
 {
     if (!_pcAction) {
         _pcAction = createAction();
+#ifdef FC_DEBUG
+        // Accelerator conflict can now be dynamically resolved in ShortcutManager
+        //
+        // printConflictingAccelerators();
+#endif

```

Comment:
No reason to leave this in
```suggestion
```

Reply:
Let's leave these there for now, because these are the only places printConflictingAccelerators() are used. They can be remove once the ShortcutManager is proven reliable.